packages/cli/src/commands/start.js

```diff
@@ -1,59 +1,21 @@
 import fs from 'fs'
 import path from 'path'
-import parseArgs from 'minimist'
 import consola from 'consola'
+import NuxtCommand from '../common/command'
 
-import { loadNuxtConfig } from '../common/utils'
-
-export default async function start() {
-  const { Nuxt } = await import('@nuxt/core')
-
-  const argv = parseArgs(process.argv.slice(2), {
-    alias: {
-      h: 'help',
-      H: 'hostname',
-      p: 'port',
-      n: 'unix-socket',
-      c: 'config-file',
-      s: 'spa',
-      u: 'universal'
-    },
-    boolean: ['h', 's', 'u'],
-    string: ['H', 'c', 'n'],
-    default: {
-      c: 'nuxt.config.js'
-    }
+export default async function build() {
```

Comment:
Shouldnt it be called start?

Reply:
Yes, you are right of course. But isnt giving that function a name useless anyway? Its the default export which is not named anyhow and the function is never re-used anywhere else

Reply:
@pimlie Named functions allows to easier tracing errors :)